users/script.js

```diff
@@ -162,10 +160,17 @@ function generateUserList(
     return;
   }
 
-  const ulElement = document.getElementById('head_list');
+  const errorTexts = document.getElementsByClassName('error-text');
+  Array.from(errorTexts).forEach((errorText) => {
+    errorText.remove();
+  });
```

Comment:
- do not select the element using classname use id instead. 
- as we know there will be only one element that will contain the error message do we need to create an array and loop over all elements that match the classname?